src/main/java/uk/gov/beis/els/categories/tumbledryers/model/rescaled/RescaledTumbleDryersForm.java

```diff
@@ -0,0 +1,185 @@
+package uk.gov.beis.els.categories.tumbledryers.model.rescaled;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.groups.Default;
+import org.hibernate.validator.constraints.Range;
+import org.hibernate.validator.group.GroupSequenceProvider;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.categories.common.StandardTemplateForm30Char;
+import uk.gov.beis.els.categories.internetlabelling.model.InternetLabellingGroup;
+import uk.gov.beis.els.categories.tumbledryers.service.RescaledTumbleDryersService;
+import uk.gov.beis.els.model.meta.DualModeField;
+import uk.gov.beis.els.model.meta.FieldPrompt;
+
+@GroupSequenceProvider(RescaledTumbleDryerFromSequenceProvider.class)
+public class RescaledTumbleDryersForm extends StandardTemplateForm30Char {
+
+  @FieldPrompt(value = "Link to the product information sheet for this product on a publicly accessible website",
+      hintText = "This link will be shown as a QR code on the label. Links should be under 300 characters to make sure they can be scanned reliably.")
+  @Pattern(regexp = "^(https|http)://([a-zA-Z0-9\\-]+)\\.[a-zA-Z0-9]+.*",
+      message = "Enter a link to the product information sheet. Links must start with http:// or https:// and contain at least one dot (.) character")
+  @NotNull
+  @Schema(description = "Enter a link to the product information sheet. Links must start with http:// or https:// and contain at least one dot (.) character")
+  private String qrCodeUrl;
+
+  @FieldPrompt("Energy efficiency class indicator")
+  @NotBlank(message = "Select an energy efficiency indicator", groups = {Default.class, InternetLabellingGroup.class})
+  @DualModeField
+  @ApiValuesFromLegislationCategory(serviceClass = RescaledTumbleDryersService.class)
+  private String efficiencyRating;
+
+  @FieldPrompt("Weighted average energy consumption per 100 drying cycles in kWh, rounded to the nearest integer")
+  @Digits(integer = 3, fraction = 0, message = "Enter the weighted average energy consumption, up to 3 digits long")
+  @Schema(type = "integer")
+  @NotNull
+  private String energyConsumptionPer100Cycles;
+  
+  @FieldPrompt("Airborne acoustic noise emission class")
```

Comment:
I changed this from the `Acoustic airborne noise emission class` prompt in your mockup to match the rest of the rescaled labels which all use `Airborne acoustic noise emission class`. Assume this is ok?